src/D2L.CodeStyle.Analyzers/Immutability/KnownImmutableTypes.cs

```diff
@@ -53,6 +53,7 @@ internal sealed class KnownImmutableTypes {
 		/// A list of known immutable special types.
 		/// </summary
 		private readonly static ImmutableArray<SpecialType> ImmutableSpecialTypes = ImmutableArray.Create(
+			SpecialType.System_Enum,

```

Comment:
Honestly, we shouldn't have removed that in the last change.  It's still a valid SpecialType that we can rule out, it just doesn't catch regular enums.  So, I'm putting it back in.